beater/sophoscentralbeat.go

```diff
@@ -237,7 +245,7 @@ func GetSophosAlerts(scb Sophoscentralbeat) (int64, error) {
 		nestedVal, _, err := scb.sophos.AlertControllerV1ImplApi.GetAlertsUsingGET1(scb.sophosAuth, scb.config.APIKey, scb.config.Authorization, scb.basepath, options)
 		if err != nil {
 			scb.logger.Error("Call to Sophos Central Server failed. Please check Credentials(authorization, api_key or header). Error : ", err)
-			return 0, err
+			return err
 		}
 
 		receivedAlertsLogCount = receivedAlertsLogCount + int64(len(nestedVal.Items))

```

Comment:
@rajanjoshi18 No need of this function if logsReceivedInCycle just near to publishing received logs